From 46611c8698f1d1e9654e6e5a6727f8601e290cc2 Mon Sep 17 00:00:00 2001
From: emancebo <emancebo@cyngn.com>
Date: Tue, 10 Feb 2015 10:26:39 -0800
Subject: [PATCH 2/2] Expose call creation time to link call recordings with
 recent call log (2/2)

Change-Id: I1fa0afa11f57b84cb53ada4de89dd5e84df60224
(cherry picked from commit 725d76f5fdb98a6cee40d8fe84016bbafc931485)
---
 src/com/android/server/telecom/ParcelableCallUtils.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/server/telecom/ParcelableCallUtils.java b/src/com/android/server/telecom/ParcelableCallUtils.java
index 21e30737..9dd99ce5 100644
--- a/src/com/android/server/telecom/ParcelableCallUtils.java
+++ b/src/com/android/server/telecom/ParcelableCallUtils.java
@@ -166,6 +166,7 @@ public class ParcelableCallUtils {
                 capabilities,
                 properties,
                 supportedAudioRoutes,
+                call.getCreationTimeMillis(),
                 connectTimeMillis,
                 handle,
                 call.getHandlePresentation(),
-- 
2.17.1

